app/javascript/lib/scp-api.js

```diff
@@ -159,15 +159,15 @@ export async function fetchCluster(
   // Digest full annotation name to enable easy validation in API
   const [annotName, annotType, annotScope] = annotation.split('--')
   const paramObj = {
-    annotation_name: annotName,
+    annotation_name: encodeURIComponent(annotName),
```

Comment:
Blocker: this would cause double-encoding bugs.  We use `queryString.stringify` to encode these parameters, and that function [strictly URI-encodes components by default](https://github.com/sindresorhus/query-string#strict).

```suggestion
    annotation_name: annotName,
```

For a demo, go to an SCP page and run the following in DevTools Console, and note the URL that gets requested.

Code on production now:
```
// Annotation name contains a slash.  Requested URL includes: bar%2Fbaz -- already URI-encoded
cluster = await SCP.API.fetchCluster('SCP1', 'foo', 'bar/baz', 'study', 'group')
```

With `annotation_name: encodeURIComponent(annotName),`:
```
// Annotation name contains a slash.  Requested URL includes: foo%252Fbar -- a double-encoding bug
cluster = await SCP.API.fetchCluster('SCP1', 'foo', 'bar/baz', 'study', 'group')
```

Reply:
Good catch!  I didn't realize that `queryString.stringify` also did this.  I can remove that function call there.  

As far as other potential double-encodings, my testing showed that this is not the case - in fact, providing a non-encoded value for `cluster_name` does not in fact load the correct cluster, and will return `404`.  I can add a negative test in `cluster_controller_test.rb` that will assert this for better coverage/confidence.

Reply:
4e950b, 0aa6082